event/register_oracle_event.go

```diff
@@ -51,9 +59,93 @@ func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
 
 	err = sgx.VerifyRemoteReport(oracleRegistration.NodePubKeyRemoteReport, nodePubKeyHash[:], *e.reactor.EnclaveInfo())
 	if err != nil {
+		msgVoteOracleRegistrationNo, err := makeOracleRegistrationVote(uniqueID, e.reactor.OracleAcc().GetAddress(), addressValue, types.VOTE_OPTION_NO, e.reactor.OraclePrivKey())
+		if err != nil {
+			return err
+		}
+
+		txBytes, err := generateTxBytes(msgVoteOracleRegistrationNo, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), e.reactor.TxBuilder())
+		if err != nil {
+			return err
+		}
+
+		err = broadCastTx(e.reactor.GRPCClient(), txBytes)
+		if err != nil {
+			return err
+		}
 		return err
 	}
 
-	// TODO: Executing Vote Txs
+	msgVoteOracleRegistrationYes, err := makeOracleRegistrationVote(uniqueID, e.reactor.OracleAcc().GetAddress(), addressValue, types.VOTE_OPTION_YES, e.reactor.OraclePrivKey())
+	if err != nil {
+		 return err
+	}
+
+	txBytes, err := generateTxBytes(msgVoteOracleRegistrationYes, e.reactor.OracleAcc().GetPrivKey(), e.reactor.Config(), e.reactor.TxBuilder())
+	if err != nil {
+		return err
+	}
+
+	err = broadCastTx(e.reactor.GRPCClient(), txBytes)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// makeOracleRegistrationVote makes a vote for oracle registration with VOTE_OPTION
+func makeOracleRegistrationVote(uniqueID, voterAddr, votingTargetAddr string, voteOption types.VoteOption, oraclePrivKey []byte) (*types.MsgVoteOracleRegistration, error) {
+	registrationVote := &types.OracleRegistrationVote{
+		UniqueId:               uniqueID,
+		VoterAddress:           voterAddr,
+		VotingTargetAddress:    votingTargetAddr,
+		VoteOption:             voteOption,
+		EncryptedOraclePrivKey: oraclePrivKey,
```

Comment:
The oracle key should be encrypted to the node public key of oracle registration.
Please encrypt it.

Reply:
Fixed in [a3ecc46](https://github.com/medibloc/panacea-doracle/pull/44/commits/a3ecc4648e60af34a710ae19e695c0fbecd2ca77).